hc/api/tests/test_admin.py

```diff
@@ -9,6 +9,9 @@ def setUp(self):
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
 
         ### Set Alice to be staff and superuser and save her :)
+        self.alice.is_staff = True
+        self.alice.superuser = True
```

Comment:
Let me make the necessary changes. That was so close, I should have seen that earlier; working on it now.